### PR TITLE
Add Ethereum Social support on multi-geth

### PR DESCRIPTION
**Pull request by ghost**
### Ethereum Social (aka ETSC) is an ethereum-fork based blockchain platform for forum sites.

You can see the info about our coin on our homepage [ethereumsocial.kr](https://ethereumsocial.kr) and korean forum site [gazua.tv](http://gazua.tv)

### Coin spec:

+ Homestead, EIP150, EIP160, ECIP1017, ECIP1041 enabled by default.
+ Block reward is 50 for era0.
+ Based on go-social node and parity.
+ Chainid and networkid is 0x1C.
+ Using the same hd wallet path with ethereum.

ps) You can see the [network stats here](https://stats.ethereumsocial.kr).

### Changes on pr:

+ Added Custom Block rewards based on ECIP-1017 ( aka SocialReward)
+ Added Ethereum Social mainnet genesis block

### CODE REVIEW

File: core/genesis.go [hunk 1]
```diff
@@ -334,7 +336,7 @@ func DefaultEllaismGenesisBlock() *Genesis {
 	}
 }
 
-// ClassicGenesisBlock returns the Ellaism genesis block.
+// ClassicGenesisBlock returns the Ethereum Classic genesis block.
```
Comment (ellaismer) [inline core/genesis.go:339]: Good catch. Thank you!

File: cmd/utils/flags.go [hunk 2]
```diff
@@ -1239,6 +1255,8 @@ func MakeGenesis(ctx *cli.Context) *core.Genesis {
 		genesis = core.DefaultEllaismGenesisBlock()
 	case ctx.GlobalBool(ClassicFlag.Name):
 		genesis = core.DefaultClassicGenesisBlock()
+  case ctx.GlobalBool(SocialFlag.Name):
```
Comment (ellaismer) [inline cmd/utils/flags.go]: Please use TAB indentation here.

**Review by ellaismer — COMMENTED** [review]
Everything else looks good. Please fix the typo mentioned above.

**Comment by ghost** [comment]
@ellaismer Will fix it, thanks!

**Comment by ghost** [comment]
@ellaismer  Fixed!

**Review by ellaismer — APPROVED** [review]

File: core/genesis.go (hunk 1 above)
Comment (ghost) [reply core/genesis.go:339]: @ellaismer ¯\_(ツ)_/¯

**Pull request merged**